packages/grafana-ui/src/components/InteractiveTable/InteractiveTable.tsx

```diff
@@ -38,12 +38,13 @@ const getStyles = (theme: GrafanaTheme2) => {
       borderRadius: theme.shape.radius.default,
       width: '100%',
 
-      td: {
-        padding: theme.spacing(1),
-      },
-
-      'td, th': {
-        minWidth: theme.spacing(3),
+      '> tbody, > thead': {
+        '> tr > td': {
+          padding: theme.spacing(1),
```

Comment:
Bug: This now overwrites the padding styles in the expandedContentRow > td. Please fix

Reply:
Removed all generic selectors that weren't already scoped with the child-combinator selector. Expanded row padding is now working as expected.